app/paid/paid_test.go

```diff
@@ -0,0 +1,74 @@
+package paid
+
+import (
+	"encoding/pem"
+	"fmt"
+	mrand "math/rand"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var testSDHash = "bea4d30a1868a00e98297cfe8cdefc1be6c141b54bea3b7c95b34a66786c22ab4e9f35ae19aa453b3630e76afbd24fe2"
+var testTxID = "e5a6a9ef4433b8868cebb770d1d2244eed410eb7503a9ffa82dce40aa62bbae7"
+
+func generateKeyFile() (string, error) {
+	keyFilename := fmt.Sprintf("test_private_key_%v.pem", time.Now().Unix())
+	cmd := fmt.Sprintf("openssl genrsa -out %s 2048", keyFilename)
+	if _, err := exec.Command("bash", "-c", cmd).Output(); err != nil {
```

Comment:
feels sketchy. id rather do this inside our code

Reply:
looks like you use `rsa.GenerateKey(rand.Reader, 2048)` in other tests. id do the same here

Reply:
I wanted this part to be as close to real world scenario as possible — it's just easier to generate the key file with openssl during deployment